geodiff/tests/geodiff_testutils.cpp

```diff
@@ -97,14 +97,23 @@ std::string tmp_file( std::string basename )
   return path;
 }
 
+static void logger( LoggerLevel level, const char *msg )
+{
+  std::string prefix;
+  switch ( level )
+  {
+    case LevelError: prefix = "err: "; break;
+    case LevelWarning: prefix = "wrn: "; break;
+    case LevelDebug: prefix = "dbg: "; break;
+    default: break;
+  }
+  std::cout << prefix << msg << std::endl ;
+}
```

Comment:
This looks like the default callback. Maybe it would be better to have the setLogging(...) function split into two so we don't need to reimplement it and only set desired logging level?